fastlane/Pluginfile

```diff
@@ -6,7 +6,7 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.9.14'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch:'ios-l10n-lint'
```

Comment:
⚠️ This needs to be reverted to point to a tag once wordpress-mobile/release-toolkit#177 is merged and we tag a new version of the release toolkit

Reply:
![image](https://user-images.githubusercontent.com/1218433/101587946-5e9fa600-3a39-11eb-81c3-dce53451656d.png)

Neat commit message reminder! I usually write "`REVERT ME`", but I like the extra color here 🚨 .

Reading this made me think "we should have a Peril rule to remind us to revert commits that start with <agreed upon revert pattern>". Then I thought a better option would be to have a Peril rule that fails the build (preventing merging) if the plugin points to anything other than a tag.

It's a reminder that we all set for ourselves when opening these PRs and that reviewers point out, too. Seems ripe for automation.